src/web/public/styles/github-dark.min.css

```diff
@@ -0,0 +1,10 @@
+pre code.hljs{display:block;overflow-x:auto;padding:1em}code.hljs{padding:3px 5px}/*!
```

Comment:
Is it possible for us to get these via `node_modules/` vs. embedding directly in our code?  Then we can get updates.

Reply:
So, as I understand how `next.js` deals with static files and CSS style sheets in particular, it might not be possible with the tools we have available...

If your style sheets were in `src/web/src/styles`, you can `@import` a style sheet found in `node_modules`, but `next.js` modifies the original file so that the `@import` rule is not there anymore. Thus, we cannot access the `github.css` and `github-dark.css` files from the `@import` rules as shown in [CSSImportRule](https://developer.mozilla.org/en-US/docs/Web/API/CSSImportRule). And, even if `next.js` kept the `@import` rule, we wouldn't be able to find the original style sheet with an `id`, because the `style` element generated by `next.js` does not have an `id`.

If you place the style sheets in `src/web/public`, you cannot `@import` a style sheet found in `node_modules`.

So, with my current understanding, it might not be possible. If someone knows a lot about `next.js`, please let us know if you can think of a way to make this possible.

Reply:
In the worst case, we would have to use a dependency that copies certain files in `node_modules` during `next.js` compilation time for this specific case.